ZgatewayRF2.ino

```diff
@@ -117,8 +113,8 @@ void MQTTtoRF2(char * topicOri, char * datacallback) {
   
   int pos = topic.lastIndexOf(RF2codeKey);       
   if (pos != -1){
-    pos = pos + strlen(RF2codeKey);
-    valueCODE = (topic.substring(pos, topic.indexOf("/", pos))).toInt();
+    pos = pos + +strlen(RF2codeKey);
+    valueCODE = (topic.substring(pos,pos + 8)).toInt();
```

Comment:
I see you added back the hard coded code length, this will make it not work with some remotes.

Reply:
indeed merge mistake, thanks for pointing this

Reply:
Also my support for the group bit and the MQTT state updating (when receiving RF commands from MQTT) didn't make it through the merge..

Reply:
ok :-(, may I ask you to repush your modifications into this branch, some remarks
just replace *Serial.println(* by *trc(*
remove *int valueBIT  = 0;*
and comment ZgatewayRF2 for arduino as the sketch is too big for Uno

Reply:
Sure, what would be the easiest way to go about going that, forking the current master branch and re-apply my changes and then make a new pull request? I'm still a newbie with git branching and pull requests.. I have also implemented support for dimming (verified to work with my Nexa WMR-250 dimmer)

Reply:
*forking the current master branch and re-apply my changes and then make a new pull request?*
I think it will be the easiest way
And we cancel this one, sorry for the rework...

Reply:
Sure, no problem